doc_source/rollup.md

```diff
@@ -49,4 +49,6 @@ GET target_index/_search
     }
   }
 }
-```
\ No newline at end of file
+```
+**Note**  
+OpenSearch 2.2 or later versions support searching multiple rollup indexes in one request. For OpenSearch versions before 2.2 and legacy Elasticsearch OSS versions, it supports only one rollup index per search.
```

Comment:
@Angie-Zhang could you make this:

OpenSearch versions 2.2 and later support searching multiple rollup indexes in one request. OpenSearch versions prior to 2.2 and legacy Elasticsearch OSS versions only support one rollup index per search.

Reply:
Updated